client/src/components/common/applicant-info-1.vue

```diff
@@ -495,7 +495,7 @@ export default class ApplicantInfo1 extends Vue {
     if (this.showAddressMenu && event.key === 'Escape') {
       this.showAddressMenu = false
     }
-    if (this.addressSuggestions && this.showAddressMenu) {
```

Comment:
This was causing "Cannot destructure property 'Id' of 'this.addressSuggestions[0]' as it is undefined."